src/redux/Products/thunk.js

```diff
@@ -0,0 +1,91 @@
+import Swal from 'sweetalert2';
+import {
+    setProducts,
+    setLoadingProducts,
+    dismissLoadingProducts,
+    getProductsError,
+    deleteProductSuccess,
+    getProductDelete
+} from './actions'
+
+const BASE_URL = 'http://localhost:4000'
+
+export function getProducts() {
```

Comment:
Handle success of this request by checking the status of the response (response.status) 
being different than 200 in this case is an error, (because is a get).